spec/http_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe CircleCi::Http do
+
+  context 'initialize' do
+
+    it 'sets the proxy server' do
```

Comment:
This test feels a little out of place for me as far as consistency goes with the rest of the tests. I'd prefer it to be written without the use of doubles and the unit test style object should receive approach. Instead there should be a real CircleCI object being created, as one would in their code, setting the proxy config, and then expecting the proxy to be set on the RestClient object after using an http object.